concent_api/conductor/tasks.py

```diff
@@ -178,7 +181,10 @@ def call_blender_verification_order() -> None:
             blender_crop_script=verification_request.blender_subtask_definition.blender_crop_script,
         )
 
-    transaction.on_commit(call_blender_verification_order, using='control')
+    transaction.on_commit(
+        call_blender_verification_order,
+        using='storage',
```

Comment:
I'm just curious why it should be `storage` database instead of `control`. Could you explain it to me? :)

Reply:
The whole task is wrapped in transaction using `storage` database.